### ISSUE TITLE
Add Custom Connector to DLP

### ISSUE THREAD

**Issue by fredericklin** [MicrosoftDocs/power-platform#564]
Would be great if there are detailed instructions on how to add the Office 365 Management API custom connector to a DLP policy.  I've tried to do this through Powershell but there is not enough documentation on this page or with the Powershell cmdlets docs as well.

* Get-AdminPowerAppConnector doesn't return anything (is it a bug?)
* Set-DlpPolicy seems to accept an "$UpdatedPolicy" object.  Would be great if there are step by step instructions for getting the object (through Get-DlpPolicy?) and adding connectors to the business and non-business groups.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1855a9ee-8713-c8e6-2091-bc972f877c13
* Version Independent ID: c87a0108-4de6-49df-8ba3-3bdb06216ad8
* Content: [Set up audit log components - Power Platform](https://docs.microsoft.com/en-us/power-platform/guidance/coe/setup-auditlog)
* Content Source: [power-platform/guidance/coe/setup-auditlog.md](https://github.com/MicrosoftDocs/power-platform/blob/master/power-platform/guidance/coe/setup-auditlog.md)
* Service: **power-platform**
* GitHub Login: @manuelap-msft
* Microsoft Alias: **mapichle**

**Comment by manuelap-msft** [comment]
Hello,

thank you for the feedback! I will share this with the DLP team to include in their area of the docs, and we can link to them from here. If they include it there, it will benefit many more customers vs us just including it in our CoE section.

Get-AdminPowerAppConnector  does not return custom connectors that are in a solution, this is a known limitation at the moment. 

In the meantime, the DLP SDK contains sample implementations of how to work with the cmdlets https://github.com/microsoft/PowerApps-Samples/tree/master/powershell/admin-center and also a sample function for how to add a custom connector to a DLP policy.

**Comment by fredericklin** [comment]
@manuelap-msft Thank you.

Since Get-AdminPowerAppConnector does not enumerate custom connectors that are in a solution, I decided to do a fiddler trace when editing the properties of the custom connector from the make.powerapps.com Custom Connectors page. That allowed me to pull the ConnectorID, ConnectorName, and ConnectorType of the Office 365 Management API custom connector.  I then used the "Add-CustomConnectorToPolicySample" sample code reference to add the custom connector to the business group.
